libs/langchain-qdrant/src/vectorstores.ts

```diff
@@ -194,6 +198,8 @@ export class QdrantVectorStore extends VectorStore {
       vector: query,
       limit: k,
       filter,
+      with_payload: [this.metadataPayloadKey, this.contentPayloadKey],
+      with_vector: false,
```

Comment:
**Improvement:** Return only required fields from the Qdrant payload. Don't return vectors of the results.